src/modules/base/navigation/menu-svc.js

```diff
@@ -53,6 +59,56 @@
             }]
         }];
 
+        vm.setMenus = function() {
```

Comment:
@gnehapk why these menus? Isn't this same as above?

Reply:
@cloudbehl Forget to remove it. Thanks.